GliaWidgets/Sources/EntryWidget/EntryWidget.MediaTypeItem.swift

```diff
@@ -0,0 +1,81 @@
+import GliaCoreSDK
+import SwiftUI
+
+extension EntryWidget {
+    struct MediaTypeItem: Equatable, Hashable {
+        let type: EngagementType
+        var badgeCount: Int
+        let headline: String
+        let subheadline: String
+        let hintline: String
+        let image: Image
+
+        static func make(type: EngagementType, badgeCount: Int = 0) -> Self {
```

Comment:
Why not use regular init?
```suggestion
        init(type: EngagementType, badgeCount: Int = 0) {
```

Reply:
No idea how I came up with that, thanks!

https://github.com/salemove/ios-sdk-widgets/pull/1112/commits/f99771f3e8f5efd277ab6d66978888c311137bea